features/src/autogluon/features/generators/datetime.py

```diff
@@ -12,15 +13,14 @@
 
 class DatetimeFeatureGenerator(AbstractFeatureGenerator):
     """Transforms datetime features into numeric features.
-
     Parameters
     ----------
     features : list, optional
         A list of datetime features to parse out of dates.
         For a full list of options see the methods inside pandas.Series.dt at https://pandas.pydata.org/docs/reference/api/pandas.Series.html
     """
-    def __init__(self, 
-            features: list = ['year', 'month', 'day', 'dayofweek'],
+    def __init__(self,
+            features: list = ['year', 'month', 'day', 'dayofweek', 'hour'],
```

Comment:
Let's not have hour be used by default, it is often just noise.

Reply:
Deleted "hour"